### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by miyazono**
New PR into master

### CODE REVIEW

**Comment by miyazono** [comment]
for the record, this had previously been approved into a different branch [here](https://github.com/sourcecred/docs/pull/25)

**Comment by vsoch** [comment]
This seems okay to me, but I'm overall really confused - @decentralion you requested that I open separate PRs per concept. I did that. What was the point of those PRs if the commits are just being now integrated into another branch for master? I'm trying really hard to follow best practices for the project and what I'm told to fix, but I'm just totally confused at this point :/

**Comment by vsoch** [comment]
Aren't this PR and https://github.com/sourcecred/docs/pull/23 now functionally the same, but for the latter we have two clean commits, here we have a bunch of WIP commits plus one clean?

**Comment by miyazono** [comment]
> This seems okay to me, but I'm overall really confused - @decentralion you requested that I open separate PRs per concept. I did that. What was the point of those PRs if the commits are just being now integrated into another branch for master? I'm trying really hard to follow best practices for the project and what I'm told to fix, but I'm just totally confused at this point :/

@vsoch, assume I'm a github noob and am probably missing signals, misinterpreting instructions, or generally failing at this

**Comment by vsoch** [comment]
@miyazono you can disregard - you’ve done great work and are following instructions too! I think the flow for how to contribute needs to be consistent, and this feels more like a wild goose chase (and I certainly don’t want to be a goose or chaser of goose) so I’m asking for clarification.

**Comment by decentralion** [comment]
Yeah, the flow I'd like to see is one PR for @miyazono's guide (with all the commits squashed into one) and one PR for the concepts in the glossary.

**Comment by vsoch** [comment]
Thanks for the clarification!

**Comment by decentralion** [comment]
Closing as the guide was merged on-its-own by #33
